source/nutrition/nutrition_facts.cpp

```diff
@@ -148,87 +138,8 @@ namespace nutrition {
         selenium -= other.selenium;
         carotene -= other.carotene;
```

Comment:
Se for usar invariantes, precisa de  `assert(invariants());` em todas as funções modificadoras.